tests/factories/runner/runner.py

```diff
@@ -0,0 +1,35 @@
+"""Test `Runner` factories."""
+import contextlib
+from unittest import mock
+
+import factory
+
+from yapapi.services import ServiceState
+
+from dapp_runner.runner import Runner
+from tests.factories.descriptor import ConfigFactory, DappDescriptorFactory
+
+
+class RunnerFactory(factory.Factory):
+    """Test factory for the dapp-runner's `Runner`."""
+
+    class Meta:  # noqa
+        model = Runner
+
+    config = factory.SubFactory(ConfigFactory)
+    dapp = factory.SubFactory(DappDescriptorFactory)
+
+    @factory.post_generation
+    def desired_app_state(obj, _, extracted: ServiceState, **__):  # noqa
+        if extracted:
+            obj._desired_app_state = extracted
+
+
+@contextlib.contextmanager
+def mock_runner(**kwargs):
```

Comment:
Consider changing this into pytest fixture and moving to `conftest.py`.
And instead of `mock` API, use `mocker` fixture for better UX.
Where usage in tests could look like this `mocked_runner = create_mocked_runner(...)`.